react/forms/step3.md

```diff
@@ -0,0 +1,77 @@
+Now we'll modify our form to include a title dropdown.
+
+## Task
+
+1) Make sure the App.js file is still open
+
+2) To create a title dropdown, we'll add a new `value1` attribute to state and a `this.options` array in the constructor.
+
+Locate the code that looks like the following:
+
+```
+constructor(props) {
+...
+}
+```
+
+Replace it with the following code:
+
+<pre class="file" data-target="clipboard">
+constructor(props) {
+  super(props);
+  this.state = {
+    value1: 'please choose',
+    value2: '',
+    value3: '',
+    value4: ''
+  };
+  this.onChange = (value1, event) => {
+    this.setState({ value1 });
+  };
+  this.handleTextInputChange1 = value2 => {
+    this.setState({ value2 });
+  };
+  this.handleTextInputChange2 = value3 => {
+    this.setState({ value3 });
+  };
+  this.handleTextInputChange3 = value4 => {
+    this.setState({ value4 });
+  };
+  this.options = [
+    { value: 'please choose', label: 'Please choose', disabled: false },
+    { value: 'miss', label: 'Miss', disabled: false },
+    { value: 'mrs', label: 'Mrs', disabled: false },
+    { value: 'ms', label: 'Ms', disabled: false },
+    { value: 'mr', label: 'Mr', disabled: false },
+    { value: 'dr', label: 'Dr', disabled: false },
+    { value: 'other', label: 'Other', disabled: false }
+  ];

```

Comment:
I'm running thru the tutorial now and I'd like to see just the new code I need to copy. This is something @dgutride and I spoke about (i.e., just highlighting the new code shows what's changed).

The 'please choose' text could be added in the previous step.